Overview.md

```diff
@@ -23,6 +23,21 @@ When the solution is just deployed, the monitoring worker does not have any acce
 Go to the deployed managed application and open the Parameters and Outputs blade. Copy the value of the monitoring Identity output.  
 < screenshot here >  
 Provide this identity with a Reader role to your subscription ([more info](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/howto-assign-access-portal))
+
+## Configure ADF Diagnostic Settings

```

Comment:
This section is like  "prerequisite", withou Diagnostic settings configuration Workbooks will not work. It's better to put it into the beginning of the guide

Reply:
Got it.